### ISSUE TITLE
Create "News" index page

### ISSUE THREAD

**Issue by elondaits** [IMAGINARY/imaginary-web#123]
Currently 3 news items are shown in the home page but there is no central index page with the full list of news items. 

Such a list is shown as a sidebar of every news item, but it's weird to have paging on a sidebar where the central item doesn't change. More importantly, on the mobile version (in development) the "All news" sidebar is shown under the item (as the layout switches to single column) and paging through items is counterintuitive.

The "News" index page could be linked from the "Imaginary News" title in the home page, a "see all news" in the sidebar (instead of the pager) and in the future it could go into a footer menu.

**Comment by HexeBianca** [comment]
your suggestion sounds good. However, I would not give it high priority, unless it is necessary for the development of the mobile page.

**Comment by elondaits** [comment]
Yes, was probably too much. It doesn't look right at all in mobile, but it's not central functionality.
